src/evaluation/factuality.py

```diff
@@ -133,6 +135,19 @@ def evaluate_summary(
             if not ent["in_source"]:
                 summary_eval["count_entity_extrinsic"] += 1
 
+        # if the summary isn't failed, then default ratio to 1.0 -- 1 then
+        # serves as the sensible "default value" for when there are no
+        # extrinsic hallucations in the summary.
+        count_entity_labels = summary_eval["count_entity_label"]
```

Comment:
set to none when there are no extrinsic hallucinations

Reply:
Addressed in https://github.com/danton-nlp/factual-beam-search/pull/108/commits/eb32985b1615a4e5215ff57627c9669a9c627972